system/lib/wasmfs/backends/opfs_backend.h

```diff
@@ -67,28 +67,27 @@ void _wasmfs_opfs_free_directory(int dir_id);
 int _wasmfs_opfs_read_access(int access_id,
                              uint8_t* buf,
                              uint32_t len,
-                             uint32_t pos);
+                             off_t pos);
 
 int _wasmfs_opfs_read_blob(em_proxying_ctx* ctx,
                            int blob_id,
                            uint8_t* buf,
                            uint32_t len,
-                           uint32_t pos,
+                           off_t pos,
                            int32_t* nread);
 
 // Synchronous write. Return the number of bytes written.
 int _wasmfs_opfs_write_access(int access_id,
                               const uint8_t* buf,
                               uint32_t len,
-                              uint32_t pos);
+                              off_t pos);
 
 // Get the size via an AccessHandle.
 void _wasmfs_opfs_get_size_access(em_proxying_ctx* ctx,
                                   int access_id,
                                   off_t* size);
 
-// TODO: return 64-byte off_t.
-uint32_t _wasmfs_opfs_get_size_blob(int blob_id);
+void _wasmfs_opfs_get_size_blob(int blob_id, off_t* size);
```

Comment:
Why can't `off_t` be used as a return type here?

Reply:
Maybe it can.  Not sure if/how  i64 values are returned, so I just used the same method as the other `get_size` function to be safe.

Reply:
I think if you mark this function as `i53abi` and `j` is the return type it should "just work".

Reply:
Good to know.  I'll keep that in mind for future updates.

Reply:
Can we make this part of this change?  No need to the out parameter then a simple return type will do (IMHO).

Reply:
Can we make this part of this change?  No need to the out parameter then a simple return type will do (IMHO).

Reply:
I've change it to return the size directly.  Seems to work fine.